src/pages/documentation/adobe-audience-manager/migration.md

```diff
@@ -1,3 +1,12 @@
+---
+title: Migrating to AEPAudience
+description: Learn how to migrate your APIs from ACPAudience to AEPAudience.
```

Comment:
Since we are referring to the SDK APIs and migrating an app implementation, can we update the other migration docs to something like the Analytics example: "Learn how to migrate from using the ACPAudience extension to the AEPAudience extension."